web/blog/mobilizing-zeek-logs/index.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Mobilizing Zeek Logs
+authors: mavam
+date: 2023-07-06
+tags: [tenzir, zeek, logs, json, kafka]
+---
+
+# Mobilizing Zeek Logs
+
+[Zeek](https://zeek.org) offers many ways to produce and consume logs. In this
+blog, we explain the various Zeek logging formats and show how you can get the
+most out of Zeek with Tenzir. We concludes with recommendations for when to use
+what Zeek format based on your use case.
+
+<!--truncate-->
+
+## Zeek Logging 101
+
+Zeek can produce output in many ways. TSV, NDSJON, and more. Let's dive right
+in.
+
+### Tab-Separated Values (TSV)
+
+When you install a stock Zeek and run `zeek` on the command line, Zeek writes
+its logs into the current directory, one per log type. Every log has its own
```

Comment:
```suggestion
its logs into the current directory, one file per log type. Every log has its own
```
But instead of the simple grammar fix I would expand on it a little. Very few tool write more than one log file, and the uninitiated reader will find that confusing.

I think something roughly like the follow should be below the `Zeek Logging 101` caption:
```
Zeek's central function is the production of Zeek log files while analyzing live traffic on a NIC or a recorded PCAP file. During a normal run, Zeek will generate an entire set of logs, each specific to a subset of the analyzed traffic. For example the connection log has general information about what is going on the transport layer, while dedicated protocol analyzers like `http`, `smb`, `snmp` and many others have their own log streams. On top of that Zeek's scripting layer allows users to install or write their own scripts that produce additional custom log streams.

In its default configuration, these logs get written into the processes working directory.
```
Then we can slightly adapt the paragraph here to make it less redundant.

Reply:
I integrated a variation of what you suggested. Thanks.

Reply:
Closing this thread. Feel free to comment on the new content for additional feedback.